packages/sdk/src/channelsWithV2Entitlements.test.ts

```diff
@@ -0,0 +1,185 @@
+/**
+ * @group with-v2-entitlements
```

Comment:
Here's a new test group for v2-only tests. Non-backwards-compatible erc1155 tests should only run on v2 spaces.